### ISSUE TITLE
storage: hard to notice a new volume group (or generally another resource, see lvolumes) that was just created

### ISSUE THREAD

**Issue by andreasn** [cockpit-project/cockpit#10059]
During a usability test on the storage page, where the participants were asked to create a volume, a lot of participants were able to create a volume group, but then failed to find the item they had created in the list.

**Comment by garrett** [comment]
Here's an HTML mockup of a possible animation to help with this issue: https://garrett.github.io/cockpit-mockweb/animation/new-row/

It uses opacity, grows in, has a highlight color and the highlight color eventually fades out. It uses squash & stretch principles (from traditional animation) and eased timing.

All one needs to do is add a class of "new" to the tbody. (I could make it work for a `<tbody>` or `<tr>`.)

**Comment by garrett** [comment]
Note: This should apply to all of Cockpit — not just volume groups. We should probably have an animations.css file that gets pulled into the base CSS.

**Comment by andreasn** [comment]
Yeah, there was an issue with participants having a hard time locating newly created logical volumes, so I think that this fix could apply there as well.

**Comment by andreasn** [comment]
I get a 404 on https://garrett.github.io/cockpit-mockweb/animation/new-row/ now it seems

**Comment by garrett** [comment]
Odd. It works @ https://garrett.github.io/cockpit-mockweb/animation/new-row (without the slash)... did GitHub change how Jekyll sites work?

**Comment by garrett** [comment]
I need to get the CSS into a library file that's used across the site, perhaps in an `animations.less` or `animations.css` file or something like that. And then the JS across the site needs to be (slightly*) adapted to use the new CSS.

_\* slightly: New rows would need to have the class of `new` (or `ct-new` or `animate-new` or whatever we decide._

**Comment by JeffMoyer** [comment]
I think the color is a bit too subtle.  Other than that, I like it, and would find it useful.

**Comment by KKoukiou** [comment]
@garrett would it be easy to paste here the ct-new class content, and then I can work on the JS side to implement it.

**Comment by KKoukiou** [comment]
so @garrett just let me know that PF is working on the design for it https://github.com/patternfly/patternfly-design/issues/985

**Comment by garrett** [comment]
I moved the repo from master to main a week or two ago and told GitHub to that the new branch to build from is main. However, GitHub never rebuilt it, so the entire repo was 404'd.

In the hopes that GitHub would rebuild it, I added a few minor commits. It finally did, so the links work again. Sorry about the breakage.

Most of these mockups are ancient, from the PF3 days. But the same idea still applies for this one, even if it looks crufty.
